doc/src/manual/arrays.md

```diff
@@ -872,10 +872,14 @@ slower than multiplication. While some arrays — like [`Array`](@ref) itself 
 are implemented using a linear chunk of memory and directly use a linear index
 in their implementations, other arrays — like [`Diagonal`](@ref) — need the
 full set of cartesian indices to do their lookup (see [`IndexStyle`](@ref) to
-introspect which is which). As such, when iterating over an entire array, it's
-much better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
-Not only will the former be much faster in cases where `A` is `IndexCartesian`,
-but it will also support [OffsetArrays](https://github.com/JuliaArrays/OffsetArrays.jl), too.
+introspect which is which). 
+
+!!! warnings
+
+    When iterating over an entire array, it's
+    much better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
+    Not only will the former be much faster in cases where `A` is `IndexCartesian`,
+    but it will also support [OffsetArrays](https://github.com/JuliaArrays/OffsetArrays.jl), too.
```

Comment:
I'm not a native English speaker, but I'm not sure "much better" fits a formal text like documentation.  Also, I'd generalise the conclusion to not make it sound `OffsetArrays` is any special, but only one instance of arrays with custom indexing:
```suggestion
    When iterating over an entire array, it is
    better to iterate over [`eachindex(A)`](@ref) instead of `1:length(A)`.
    Not only will the former be much faster in cases where `A` is `IndexCartesian`,
    but it will also support arrays with custom indexing, such as [OffsetArrays](https://github.com/JuliaArrays/OffsetArrays.jl).
```